sabnzbd/utils/file_extension.py

```diff
@@ -293,7 +300,7 @@ def what_is_most_likely_extension(file_path: str) -> str:
 
     for possible_extension in all_possible_extensions(file_path):
         # let's see if technically-suggested extension by puremagic is also likely IRL
-        if possible_extension in ALL_EXT:
+        if possible_extension in all_extensions():
```

Comment:
The new function is now called again and again for every iteration of the for loop.
Maybe assign it to a variable before the loop?